Jenkinsfile

```diff
@@ -85,7 +89,7 @@ pipeline {
                   withCredentials([string(credentialsId: 'hide-server-url', variable: 'TOKEN')]) {
                     sh "sed -i -e \"s,$TOKEN,,g\" avscan_infomodel/target/inl-releng-avsupport/avscan_report.html"
                   }
-                      s3Upload(file:'avscan_infomodel/target/inl-releng-avsupport/avscan_report.html', bucket:'pr-vorto-documents', path:"avscans/${CHANGE_ID}/${BUILD_NUMBER}/infomodelrepository_report.html")
+                      // s3Upload(file:'avscan_infomodel/target/inl-releng-avsupport/avscan_report.html', bucket:'pr-vorto-documents', path:"avscans/${CHANGE_ID}/${BUILD_NUMBER}/infomodelrepository_report.html")
```

Comment:
Why u comment out the s3 upload ?

Reply:
So i can uncomment after I have a solution why it doesn't work at a later time. Today all PR's are getting blocked due to this.